extension/test/test-logger.js

```diff
@@ -10,7 +10,13 @@ var verifyAsyncOutput = function (test, logger, testFunc) {
   var message = '';
 
   return function (status) {
-    var line = {}, hasMore;
+    var line = {}, hasMore, isSuccessStatus;
+
+    isSuccessStatus = typeof status == "number";
+    test.assert(isSuccessStatus, status);
+    if (!isSuccessStatus) {
+      return;
+    }
```

Comment:
So what do you actually want to test here? I have to admit that I don't know anymore what the status parameter is here. Do you want to check if it is a number and if not return early? If that is the case it should be enough to use ´assert.ok(typeof status == "number")´.